src/Pages/settings.jsx

```diff
@@ -8,6 +8,10 @@ function Settings() {
         navigate("/home");
     };
 
+    // const handleSignout = () => {
+    //     navigate("/authPage")
+    // }
```

Comment:
Keep this active for now, so we can see the page through this button
`navigate("/auth")`